tests/app/test_internal_user_service.py

```diff
@@ -28,6 +47,22 @@ def test_results_default_information_returned_for_group_user(self):
 
         self.assertEqual(expected, actual)
 
+    @requests_mock.mock()
+    def test_default_user_returned_if_http_error_non_404(self, mock_request):
```

Comment:
So the test name should reflect the actions of the test, default_user is not return on this one, it has raised a httperror, ergo test should be something like ``` test_http_error_raised_on_user_401 ``` you can use not 404 I guess, but if you read the code thats not quite right

Reply:
Agreed. This has been updated